### ISSUE TITLE
Update 'help' behavior with recent additions

### ISSUE THREAD

**Issue by jywarren** [publiclab/plotsbot#86]
As mentioned in comments of #69, let's update the 'help' behavior to reflect the full set of possible commands.

But is it statically defined? Couldn't we have it dynamically find and describe behaviors?

**Comment by ryzokuken** [comment]
Sounds great! What if we also had a `helpText` entry inside each behavior and the person who wrote the behavior also wrote the help text for that behavior?

**Comment by jywarren** [comment]
That sounds super. A little like how we're trying to make the `info.json`
descriptions in image-sequencer modules

**Comment by ryzokuken** [comment]
Exactly. I'll make an issue for that as well. Perhaps we could convert it into a bunch of FTOs instead (for adding a help text inside each current behavior).
